lib/common/ResourcePath.cpp

```diff
@@ -128,6 +132,18 @@ std::string Wasp::GetPythonDir()
     return path;
 }
 
+void Wasp::SetHDF5PluginPath() {
+    string plugins = Wasp::GetResourcePath("share/plugins");
+
+#ifndef WIN32
+    H5PLreplace(plugins.c_str(), 0);
+#else
+    plugins = "HDF5_PLUGIN_PATH=" + plugins;
+    int rc=_putenv(plugins.c_str());
+    if (rc != 0) MyBase::SetDiagMsg("Unable to set environtment variable %s", plugins.c_str());
```

Comment:
Should this be a warning rather than a diagnostic message? I don't think the user ever sees our diagnostic messages.

Reply:
Good point.  Fixed.